web/src/components/drawer/record/record-panel.tsx

```diff
@@ -82,7 +82,13 @@ export const RecordPanel: React.FC<RecordDrawerProps> = ({
     const forbiddenColumns = [ColumnsId.ifdirs, ColumnsId.interfaces];
     return columns.filter((c: Column) => {
       const value = c.value(record);
-      return !forbiddenColumns.includes(c.id) && value !== null && value !== '' && !Number.isNaN(value);
+      return (
+        !forbiddenColumns.includes(c.id) &&
+        value !== undefined &&
```

Comment:
I think it was on purepose to keep `undefined` values here.

The goal was having `null` to skip a column display when misconfigured and undefined when the data is missing from the record which will result in empty text display

Reply:
ok I missed this subtelty and it's probably why I have some tests failing.
I'd like somehow to make this more obvious ... but not sure how